lib/challenge.js

```diff
@@ -60,7 +80,18 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
-
+var Pet = function Pet (petName)
+{
+  this.petName = name;
+  this.species = species;
+  this.breed = breed;
+  this.petSound = petSound []
```

Comment:
This line is syntactically incorrect. Can you explain why? Come see an instructor or DIR to get on track with this.